config/tall-forms.php

```diff
@@ -28,6 +28,8 @@
     'multiselect-placeholder' => 'global.multiselect_placeholder', //'Please select one or multiple options ...'
     'multiselect-help' => 'global.multiselect_help', //'Press CTRL(Windows) or CMD(Mac), to select/deselect multiple options.'
 
+    //Choice placeholders and help
+    'choice-placeholder' => 'global.choice_placeholder', //'Please choice one or multiple options ...'
```

Comment:
change the comment to

//'Please select one or multiple options ...'

Reply:
Okay